tests/integration/test_integration.py

```diff
@@ -746,20 +747,27 @@ def test_downgrade(encoding, monkeypatch):
 
 
 @allencodings
-def test_span_tags(encoding, monkeypatch):
-    monkeypatch.setenv("DD_TRACE_API_VERSION", encoding)
+@pytest.mark.snapshot()
+def test_span_tags(encoding, ddtrace_run_python_code_in_subprocess):
+    import os 
```

Comment:
move to top of file, no reason it needs to be here.

Reply:
It's already at the top. I don't know why I left it in. 😓